src/wmnds/patterns/footer/_footer.scss

```
@@ -21,4 +21,169 @@
       display: block;
     }
   }
+
+  // footer v2 only styles
+  &--v2 {
+    padding: $size-lg $size-xs;
+
+    .wmnds-btn {
+      display: block;
+      color: get-color(primary);
+      background-color: $white;
+    }
+
+    .wmnds-footer__link {
+      color: currentColor;
+      font-weight: normal;
+      text-decoration: none;
+
+      &:hover {
+        color: currentColor;
+        text-decoration: underline;
+      }
+    }
+  }
+
+  @media screen and (max-width: $breakpoint-md - 1) {
+    .wmnds-footer__updates-col {
+      padding-bottom: $size-md * 1.25;
+
+      &:not(:last-child) {
+        margin-bottom: $size-xsm;
+        border-bottom: 1px solid get-color(secondary, 70);
+      }
+
+      p {
+        font-size: 0.937rem;
+      }
+    }
+  }
+
+  .wmnds-footer__heading {
+    margin-bottom: $size-md * 1.25;
+    padding-bottom: $size-md * 1.125;
+    border-bottom: 1px solid get-color(secondary, 70);
+
+    @media screen and (max-width: $breakpoint-md - 1) {
+      margin-bottom: $size-md;
+      font-weight: normal;
+
+      &.wmnds-collapse-heading {
+        margin: 0;
+        padding: $size-xsm 0;
+        border: 0;
+        font-family: $x-font-family;
+        font-size: $size-md * 1.125;
+      }
+    }
+
+    @media screen and (min-width: $breakpoint-md) {
+      padding-bottom: $size-md * 1.25;
+    }
+  }
+
+  //
+  .wmnds-footer__menu {
+    margin: 0;
+    padding: 0;
+
+    li {
+      margin-bottom: $size-sm;
+      list-style: none;
+    }
+  }
+
+  @media screen and (min-width: $breakpoint-md) {
+    .wmnds-grid {
+      justify-content: space-between;
+    }
+
+    .wmnds-col-lg-1-3 {
+      max-width: 30%;
```

I would do this with padding instead so we are not overwriting the grid.
```suggestion
      padding: 0 $size-xsm;
```